typegraph/node/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@typegraph/sdk",
+  "version": "0.3.1",
```

Comment:
likely dev/lock or dev/update needs to be adjusted to upgrade the version

Reply:
The version and description are already automatically updated by the dev/lock.